src/execution/operator/projection/physical_tableinout_function.cpp

```diff
@@ -52,14 +54,30 @@ unique_ptr<GlobalOperatorState> PhysicalTableInOutFunction::GetGlobalOperatorSta
 	return std::move(result);
 }
 
+void PhysicalTableInOutFunction::PrepareOrdinality(DataChunk &chunk, idx_t &ord_index, bool &ord_reset) const {
+	const idx_t ordinality = chunk.size();
+	if (ordinality > 0) {
+		if (ord_reset) {
+			ord_index = 1;
+			ord_reset = false;
+		}
+		const idx_t ordinality_column = column_ids.size() - 1;
+		chunk.data[ordinality_column].Sequence(ord_index,1, ordinality);
```

Comment:
Can we please verify that ord_index < ordinality

Reply:
> The third argument (ordinality) had me confused at first as it does not tell the function where to end the Sequence but instead is the amount of steps to be taken.
Ah that might be the cause of my confusion then, I jumped the gun then, ignore my comment :)